api/src/main/java/com/datastrato/gravitino/authorization/Privilege.java

```diff
@@ -21,7 +21,67 @@ public interface Privilege {
 
   /** The name of this privilege. */
   enum Name {
-    /** The privilege of load a catalog. */
-    LOAD_CATALOG
+    /** The privilege to list catalogs. */
+    LIST_CATALOG,
+    /** The privilege to load a catalog. */
+    LOAD_CATALOG,
+    /** The privilege to create a catalog. */
+    CREATE_CATALOG,
+    /** The privilege to alter a catalog. */
+    ALTER_CATALOG,
+    /** The privilege to drop a catalog. */
+    DROP_CATALOG,
+    /** The privilege to list schemas. */
+    LIST_SCHEMA,
+    /** The privilege to load a schema. */
+    LOAD_SCHEMA,
+    /** The privilege to create a schema. */
+    CREATE_SCHEMA,
+    /** The privilege to alter a schema. */
+    ALTER_SCHEMA,
+    /** The privilege to drop a schema. */
+    DROP_SCHEMA,
+    /** The privilege to list tables. */
+    LIST_TABLE,
+    /** The privilege to load a table. */
+    LOAD_TABLE,
+    /** The privilege to alter a table. */
+    ALTER_TABLE,
+    /** The privilege to create a table. */
+    CREATE_TABLE,
+    /** The privilege to drop a table. */
+    DROP_TABLE,
+    /** The privilege to read a table. */
+    READ_TABLE,
+    /** The privilege to write a table. */
+    WRITE_TABLE,
+    /** The privilege to list filesets. */
+    LIST_FILESET,
+    /** The privilege to load a fileset. */
+    LOAD_FILESET,
+    /** The privilege to alter a fileset. */
+    ALTER_FILESET,
+    /** The privilege to create a fileset. */
+    CREATE_FILESET,
+    /** The privilege to drop a fileset. */
+    DROP_FILESET,
+    /** The privilege to read a fileset. */
+    READ_FILESET,
+    /** The privilege to write a fileset. */
+    WRITE_FILESET,
+    /** The privilege to list topics. */
+    LIST_TOPIC,
+    /** The privilege to load a topic. */
+    LOAD_TOPIC,
+    /** The privilege to create a topic. */
+    CREATE_TOPIC,
+    /** The privilege to drop a topic. */
+    DROP_TOPIC,
+    /** The privilege to alter a topic. */
+    ALTER_TOPIC,
+    /** The privilege to read a topic. */
+    READ_TOPIC,
+    /** The privilege to write a topic. */
+    WRITE_TOPIC
```

Comment:
The name here are too specific for metadata, users may not know that is it, like LOAD? I think you'd better have a more suitable name for users easy to understand.

Reply:
LOAD_TOPIC means that we read the meta data of topic while READ_TOPIC means that we read data of topic.